test/test_launch_daemons.py

```diff
@@ -109,11 +107,79 @@ def test_arbiter_bad_configuration(self):
         sleep(5)
 
         ret = arbiter.poll()
+        print("*** Arbiter exited with code: %d" % ret)
         assert ret is not None, "Arbiter is still running!"
+        # Arbiter process must exit with a return code == 2
+        assert ret == 2
+
+    def test_arbiter_bad_configuration(self):
+        """ Running the Alignak Arbiter with bad configuration (unknown file or dir)
+
+        :return:
+        """
+        # copy etc config files in test/cfg/run_test_launch_daemons and change folder
+        # in the files for pid and log files
+        if os.path.exists('./cfg/run_test_launch_daemons'):
+            shutil.rmtree('./cfg/run_test_launch_daemons')
+
+        shutil.copytree('../etc', './cfg/run_test_launch_daemons')
+        files = ['cfg/run_test_launch_daemons/daemons/arbiterd.ini',
+                 'cfg/run_test_launch_daemons/arbiter/daemons/arbiter-master.cfg']
+        replacements = {
+            '/usr/local/var/run/alignak': '/tmp',
```

Comment:
Bsically we prefere regenerating config over have the conf specific for the test? This will help to see a config change break but make test harder to read.

Could we factorize this in a "util" function in alignak_test or something?